_locales/en/messages.json

```diff
@@ -1386,9 +1386,12 @@
 	"hub_home_header_checkbox_label": {
 		"message": "Share additional anonymous analytics data to improve Ghostery’s performance."
 	},
-	"hub_home_header_info": {
-		"message": "New users are opted in to participating in Human Web and Ghostery Rewards. You may change these settings in custom setup or in the extension. "
+	"hub_home_header_info_opt_in": {
```

Comment:
I know this is nitpicky, but can we call this `hub_home_header_info_opted_out` instead? Just because the key difference between this and the `hub_home_header_info` string is that the former says "New users are opted out of particpating..." in the first sentence.

Reply:
done